pkg/util/steps/runner.go

```diff
@@ -54,6 +57,16 @@ func Run(ctx context.Context, log *logrus.Entry, pollInterval time.Duration, ste
 
 		if err != nil {
 			log.Errorf("step %s encountered error: %s", step, err.Error())
+			if strings.Contains(err.Error(), "ThrottlingLimitExceeded") || strings.Contains(err.Error(), "TooManyRequests") {
+				correlation_id := fmt.Sprintf("%v", log.Data["correlation_id"])
+				message := fmt.Sprintf("Requests are being throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot#exceeding-azure-storage-limits for more details. CorrelationId: " + correlation_id)
```

Comment:
```suggestion
				message := fmt.Sprintf("Requests are throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot#exceeding-azure-storage-limits for more details. CorrelationId: " + correlation_id)
```

Reply:
Any details on why the error is happening should be within the link. I think this is too wordy

Reply:
`Requests are being throttled` is colloquial english. So that was fine wording. To shorten it you might be able to do something like `Requests Throttled. Visit () for more info. CorrelationId: `

Reply:
We generated this error message with reference to the original message that we go from Azure errors : "The request is being throttled as the limit has been reached for operation type - Read_ObservationWindow_00:05:00. For more information, see - https://aka.ms/srpthrottlinglimits"